src/plugins/base64/CMakeLists.txt

```diff
@@ -7,6 +7,6 @@ add_plugin (base64
 		    base64.c
 	    COMPILE_DEFINITIONS ELEKTRA_PLUGIN_NAME=\"base64\"
 				ELEKTRA_PLUGIN_NAME_C=base64
-	    ADD_TEST
 	    TEST_README
+	    ADD_TEST
```

Comment:
Is this change cosmetic (I agree it is nice to have the same order across files) or needed?

Reply:
This change was introduced, because I disabled the unit test in the PR, and did not insert `ADD_TEST` at the same location after I re-enabled the test.